butter-cms.json

```diff
@@ -1,23 +1,23 @@
 {
     "Timestamp": "2018-08-01T13:38:27.000Z",
     "Name": "ButterCMS",
-    "Version": "06-2019",
+    "Version": "05-2020",
     "License": "Proprietary",
     "Inception": "05-2015",
-    "Category": "Essential",
+    "Category": "Essential - Professional - Enterprise",
     "Open Source": "No",
     "Cloud Service": "Yes",
     "On Premises Installation": "No",
-    "Cloud Service Hosted in Europe": "?",
+    "Cloud Service Hosted in Europe": "No",
     "Commercial Support Available?": "Yes",
-    "GraphQL API": "No",
+    "GraphQL API": "Yes with Gatsby",
     "GraphQL [Mutations]": "No",
     "GraphQL [Subscriptions]": "No",
     "REST API": "Yes",
     "REST [Create]": "Yes",
     "REST [Read]": "Yes",
     "REST [Update]": "Yes",
-    "REST [Delete]": "No",
+    "REST [Delete]": "Yes",
```

Comment:
https://buttercms.com/docs/api/

Reply:
I did not find the delete REST endpoint.

Reply:
Working on documenting this one and will get back to you.

Reply:
We can update this in a follow up PR once we're ready.

Reply:
```suggestion
    "REST [Delete]": "No",
```